R/archivr.R

```diff
@@ -259,7 +260,7 @@ view_archiv <- function (lst, method="wayback") {
 #' @return a dataframe containing the url, status, availability,
 #'   archived url(s) and timestamp(s)
 #' @examples
-#' view_archiv.fromUrl("https://www-cs-faculty.stanford.edu/~knuth/retd.html", method="both")
+#' checkArchiveStatus <- view_archiv.fromUrl("https://www-cs-faculty.stanford.edu/~knuth/retd.html", method="both")
```

Comment:
Checkreview complained about the example line being over 100 characters.  I suggest:

``` 
#' checkArchiveStatus <- view_archiv.fromUrl(
#'     "https (..)",
#'     method="both")